documentation/book/cluster-operator.adoc

```diff
@@ -515,6 +518,27 @@ Furthermore, containers doing lots of disk I/O (such as Kafka broker containers)
 for use as operating system page cache. On such containers, the request memory should be substantially more than the
 memory used by the JVM.
 
+[[kafka_rack]]
+===== Kafka rack
+
+It is possible to enable Kafka rack-awareness by specifying a JSON string for the `kafka-rack` field in the cluster ConfigMap.
+Such a JSON object has one mandatory field named `topologyKey`.
```

Comment:
I don't know how @tombentley or @kyguy feel about this. The _Such a JSON object_ sounds really strange to me. Why not say something like _The `kafka-rack` JSON object_?